xmodule/modulestore/tests/test_mongo.py

```diff
@@ -812,10 +829,12 @@ def _check_delete_key_error(self, key):
             self.kvs.get(key)
         assert not self.kvs.has(key)
 
-    def test_delete(self):
-        yield (self._check_delete_key_error, KeyValueStore.Key(Scope.content, None, None, 'foo'))
-        yield (self._check_delete_default, KeyValueStore.Key(Scope.children, None, None, 'children'), [])
-        yield (self._check_delete_key_error, KeyValueStore.Key(Scope.settings, None, None, 'meta'))
+    @pytest.mark.parametrize('expected', ['foo', 'meta'])
+    def test_delete_key_error(self, expected):
+        self.self._check_delete_key_error(KeyValueStore.Key(Scope.content, None, None, expected))
```

Comment:
Hi @Agrendalath , thank you for the feedback, I will look into it.
Meanwhile, can you tell me how to set up my devstack so that I could run unit tests ? I followed the instruction on running the stack using `tutor` but couldn't find any information on how to run some of the pipelines locally.

Reply:
@NguyenTuanAnh2195, I don't use `tutor`, but you could try the [Running edx-platform unit tests](https://docs.tutor.overhang.io/dev.html#running-edx-platform-unit-tests) section from its docs.

If it doesn't work, you can use [these instructions](https://edx.readthedocs.io/projects/open-edx-devstack/en/latest/getting_started.html) to set up the devstack and use [this approach](https://edx.readthedocs.io/projects/open-edx-devstack/en/latest/testing_and_debugging.html) to run tests there.

Reply:
@Agrendalath Thank you so much for these resources, I will look into them and get back to you.